<launch>        
  <!-- 
       Start an apriltag node using the default settings
       to connect to an OpenNI device (e.g. Kinect). 
  -->
  <node pkg="apriltag_kinect2" type="apriltag_kinect2" name="apriltag_kinect2" output="screen">
    <param name="~default_tag_size" value="0.037" />

    <param name="~viewer" value="false" />
    <param name="~publish_detections_image" value="true" />

    <param name="~display_type" value="CUBE" />
    <param name="~marker_thickness" value="0.02" />

    <remap from="~image" to="/kinect2_victor_head/hd/points"/>
    <remap from="~camera_info" to="/kinect2_victor_head/hd/camera_info"/>
    <remap from="~marker_array" to="/apriltag_kinect2/marker_array"/>
    <remap from="~detections" to="/apriltag_kinect2/detections"/>

  </node>
</launch>
